docs/apis/scopes.md

```diff
@@ -3,70 +3,35 @@ title: Scopes
-- `resource-servers:update`: **TODO**: Description of the scope
-- `resource-servers:delete`: **TODO**: Description of the scope
-
-### API Tokens
-- `tokens:create`: **TODO**: Description of the scope
-- `tokens:read`: **TODO**: Description of the scope
-- `tokens:update`: **TODO**: Description of the scope
-- `tokens:delete`: **TODO**: Description of the scope
-- `tokens:introspect`: **TODO**: Description of the scope
-
-### Credentials
-- `credentials:read`: **TODO**: Description of the scope
-- `credentials:revoke`: **TODO**: Description of the scope
-- `credential-binding-jobs:create`: **TODO**: Description of the scope
-- `credential-binding-jobs:read`: **TODO**: Description of the scope
-
-### Themes
-- `themes:create`: **TODO**: Description of the scope
-- `themes:read`: **TODO**: Description of the scope
-- `themes:update`: **TODO**: Description of the scope
-
-### Events
-- `events:read`: **TODO**: Description of the scope
-
-### Console Configs
-- `console-configs:read`: **TODO**: Description of the scope
-- `console-configs:update`: **TODO**: Description of the scope
+Requests to the Beyond Identity Management API must be authorized and contain appropriate scopes for the actions being taken. The scopes required for a given request are intended to be intuitive based on the API being called. They are formatted as `<resource>:<action>`, where `resource` will map exactly to the resource being requested (in the URL), and `action` will be a CRUD action (Create, Read, Update, Delete) that can be performed on that resource.
+
+For example, the following curl request would require the scope `applications:read`:
+
+```
+curl https://api-us.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/applications/$APPLICATION_ID \
+  -X GET \
+  -H "Authorization": "Bearer: $TOKEN" \
+```
+
+## Scope Types
+
+- `<resource>:read`
+	- Allows for the retrieval of the resource information via a GET request to `/v1/tenants/$TENANT_ID/realms/$REALM_ID/<resource>/$RESOURCE_ID`.
+		- For realms, paths omit `/<resource>/$RESOURCE_ID`
+		- For tenants, paths omit `/realms/$REALM_ID/<resource>/$RESOURCE_ID`
+	- This same scope also gates requests to list more than one of the resource via a GET request to `/v1/tenants/$TENANT_ID/realms/$REALM_ID/<resource>`. 
+	- Requests to list multiple resources will always be filtered to the tenant and realm for which they are requested.
+
+- `<resource>:create`
+	- Allows for the creation of a new resource via a POST request to `/v1/tenants/$TENANT_ID/realms/$REALM_ID/<resource>`.
+	- Resource creation may involve the auto-population of one or many fields on the resource itself. At a minimum, this will include an identifier (`id`) field. Such fields would not be allowable to be sent on creation but will be returned in the API response. Documentation of which fields are available for which resources is available in our [API documentation](https://developer-docs-git-v1-beyondidentity.vercel.app/api/v1).
```

Comment:
How do I get this programmatically?

Reply:
@mollykarcher you can just link to (`/api/v1`). Docusaurus does the completion.